modin/pandas/test/test_expanding.py

```diff
@@ -0,0 +1,128 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pytest
+import numpy as np
+import pandas
+import modin.pandas as pd
+
+from .utils import (
+    df_equals,
+    test_data_values,
+    test_data_keys,
+    eval_general,
+    create_test_dfs,
+)
+from modin.config import NPartitions
+
+NPartitions.put(4)
+
+
+def create_test_series(vals):
+    if isinstance(vals, dict):
+        modin_series = pd.Series(vals[next(iter(vals.keys()))])
+        pandas_series = pandas.Series(vals[next(iter(vals.keys()))])
+    else:
+        modin_series = pd.Series(vals)
+        pandas_series = pandas.Series(vals)
+    return modin_series, pandas_series
+
+
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+@pytest.mark.parametrize("min_periods", [None, 5])
+@pytest.mark.parametrize("axis", [0, 1])
+@pytest.mark.parametrize(
+    "method, kwargs",
+    [
+        ("count", {}),
+        ("sum", {}),
+        ("mean", {}),
+        ("var", {"ddof": 0}),
+        ("std", {"ddof": 0}),
+        ("min", {}),
+        ("max", {}),
```

Comment:
Some of the interfaces are not tested, e.g., `skew` and `kurt`. See my comment above.